src/Plugins/Wotd.php

```diff
@@ -209,8 +242,15 @@ private function postWotdMessageInRoom(ChatRoom $room, bool $addTag): Promise
     {
         return \Amp\resolve(function() use($room, $addTag) {
             $response = yield $this->httpClient->request(self::API_URL);
-
-            return yield $this->chatClient->postMessage($room, $this->getMessage($response, $addTag));
+			
+			$message = $this->getMessage($response, $addTag);
+			
+			if($message === "") {
```

Comment:
As said I think I prefer throwing an exception or at the very least return `null` instead of an empty string.